src/Tools/GetDocumentInsider/src/Resources.resx

```diff
@@ -194,4 +194,16 @@
   <data name="QuietDescription" xml:space="preserve">
     <value>Suppresses all output except warnings and errors.</value>
   </data>
-</root>
\ No newline at end of file
+  <data name="OpenApiVersionDescription" xml:space="preserve">
+    <value>The OpenAPI spec version to use when generating the document. Optional.</value>
+  </data>
+  <data name="DocumentNameDescription" xml:space="preserve">
+    <value>The name of the OpenAPI document to generate. Optional.</value>
+  </data>
+  <data name="InvalidOpenApiVersion" xml:space="preserve">
+    <value>Invalid OpenAPI spec version '{0}' provided. Falling back to default: v3.1.</value>
```

Comment:
The doc comment says `OpenApi3_0`, but the resource string says "3.1".  Are they equivalent?

Reply:
No 😓 will fix